docs/pages/deploy-a-cluster/helm-deployments/azure.mdx

```diff
@@ -0,0 +1,335 @@
+```
+
+## Step 4/5. Set up DNS
+
+We'll now set up DNS `A` records for `teleport.example.com` and
+`*.teleport.example.com`, using your Azure DNS zone rooted at `example.com`. If
+you're using a different DNS hosting service, follow their instructions instead.
+
+```code
+$ PREFIX=teleport
+$ ZONE=example.com
+$ ZONE_RG=dns-rg
+$ external_ip="$(kubectl --namespace <Var name="teleport-ns" /> get service/<Var name="teleport-release" /> -o jsonpath='{.status.loadBalancer.ingress[*].ip}')"
+$ az network dns record-set a add-record --resource-group ${ZONE_RG} --zone-name ${ZONE} --record-set-name ${PREFIX} --ipv4-address "${external_ip}"
+$ az network dns record-set a add-record --resource-group ${ZONE_RG} --zone-name ${ZONE} --record-set-name "*.${PREFIX}" --ipv4-address "${external_ip}"
+```
+
+## Step 5/5. Create a Teleport user
+
+Create a user to be able to log into Teleport. This needs to be done on the Teleport auth server,
+so we can run the command using `kubectl`:
+
+<Tabs>
+<TabItem scope={["oss"]} label="Open Source">
+
+```code
+$ kubectl --namespace <Var name="teleport-ns" /> exec deploy/<Var name="teleport-release" />-auth -- tctl users add test --roles=access,editor
+
+User "test" has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h:
+https://teleport.example.com:443/web/invite/91cfbd08bc89122275006e48b516cc68
+
+NOTE: Make sure teleport.example.com:443 points at a Teleport proxy that users can access.
+```
+
+</TabItem>
+<TabItem scope={["enterprise"]} label="Enterprise">
+
+```code
+$ kubectl --namespace <Var name="teleport-ns" /> exec deploy/<Var name="teleport-release" />-auth -- tctl users add test --roles=access,editor,reviewer
+
+User "test" has been created but requires a password. Share this URL with the user to complete user setup, link is valid for 1h:
+https://teleport.example.com:443/web/invite/91cfbd08bc89122275006e48b516cc68
+
+NOTE: Make sure teleport.example.com:443 points at a Teleport proxy that users can access.
+```
+
+</TabItem>
+</Tabs>
+
+Load the user creation link to create a password and set up 2-factor authentication for the Teleport user via the web UI.
+
+### High Availability
+
+In this guide, we have configured 2 replicas. This can be changed after cluster creation by altering the `highAvailability.replicaCount`
+value [using `helm upgrade` as detailed below](#upgrading-the-cluster-after-deployment).
+
+## Upgrading the cluster after deployment
+
+To make changes to your Teleport cluster after deployment, you can use `helm upgrade`.
+
+Helm defaults to using the latest version of the chart available in the repo, which will also correspond to the latest
+version of Teleport. You can make sure that the repo is up to date by running `helm repo update`.
+
+If you want to use a different version of Teleport, set the [`teleportVersionOverride`](../../reference/helm-reference/teleport-cluster.mdx#teleportversionoverride) value.
```

Comment:
We should advise people to use `--version` instead to keep the deployed chart version in sync with the deployed Teleport version.

```suggestion
If you want to use a different version of Teleport, pass the `--version` argument to Helm:

```code
$ helm upgrade --version 14.0.0 \
  <Var name="teleport-release" /> teleport/teleport-cluster \
  --namespace <Var name="teleport-ns" /> \
  -f azure-values.yaml
```